package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "qless-js",
+  "name": "qless-js-inst",
   "version": "0.4.10",
```

Comment:
Um... why is `version` in `package-lock.json` different?

Reply:
fixed, thank you